_sass/about.scss

```diff
@@ -0,0 +1,24 @@
+.councils {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  .council {
+    margin-bottom: 1.5rem;
+
+    &__name {
+      font-weight: bold;
+      font-size: 1.25em;
+    }
+    &__description {
+      font-size: 1.1em;
+    }
```

Comment:
The tracking of the original font is a bit tight, perhaps adjust it? e.g. the bios use `letter-spacing: 0.5px`